app/mysql/post.js

```diff
@@ -0,0 +1,52 @@
+module.exports = {
+
+/* lgin-user */
+  checkUser: function(res, req, con) {
+    
+    var email = req.body.email;
+    
+    con.query("SELECT COUNT(*) AS occur FROM users WHERE email = ?;", email, function(err, count) {
+      if(err)
+        res.send(err.message);
+    
+      res.json(count["0"].occur);
+      return;
+    });
+  },
+  
+/* lgin-pswd-02 */
```

Comment:
There's a file in the main directory called "partial tag-list". This was an attempt to be organized so that it could be referenced to understand what was going on.

The ones underneath that have a dash in front of them ( -02 ) mean that this part of the code is a sub-subject of the main tag (like lgin-pswd-02).

So if you look at the -02 underneath lgin-pswd-02, and you went to the tag-list file and followed the bullets to lgin-pswd-02-02, that would be the area I was working on.